doc/classes/EditorExportPlatform.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorExportPlatform" inherits="RefCounted" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
```

Comment:
Missing docs.

Reply:
Not much to document yet, this is likely WIP since it should have GDVIRTUALS at some point to allow extensions to create their own export platforms.